docs/getting_started/slurm.md

```diff
@@ -0,0 +1,122 @@
+---
+icon: material/server-network
+---
+
+# 🔌 **Running on a SLURM cluster**
+
+_This documentation is a work in progress._
+
+OmniGibson can be run on a SLURM cluster using the _enroot_ container software, which is a replacement
+for Docker that allows containers to be run as the current user rather than as root. _enroot_ needs
+to be installed on your SLURM cluster by an administrator.
+
+With enroot installed, you can follow the below steps to run OmniGibson on SLURM: 
+
+1. Download the dataset to a location that is accessible by cluster nodes. To do this, you can use
+the download_dataset.py script inside OmniGibson's scripts directory, and move it to the right spot
+later. **This step is already done for SVL and Viscam nodes**
```

Comment:
this seems like internal users? maybe preface this with something like `For Stanford users:`